_posts/en/newsletters/2020-11-11-newsletter.md

```diff
@@ -0,0 +1,144 @@
+---
+title: 'Bitcoin Optech Newsletter #123'
+permalink: /en/newsletters/2020/11/11/
+name: 2020-11-11-newsletter
+slug: 2020-11-11-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter shares the announcement of a marketplace for
+incoming LN channels Also included are our regular sections with summaries of a
+Bitcoin Core PR Review Club meeting and notable changes to popular Bitcoin
+infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Incoming channel marketplace:** Olaoluwa Osuntokun [announced][pool
+  announce] a new marketplace for buying incoming LN channels.  Users and
+  merchants need channels with incoming capacity in order to be able to
+  quickly receive funds over LN.  Some existing node operators already
+  provide incoming channels, either for free or as a paid service, but
+  the Lightning Pool marketplace hopes to make this service more
+  standardized and competitive.  The initial focus is a contract for
+  [highly ranked][node rankings] node operators to provide an incoming
```

Comment:
(feel free to ignore) I think the node rankings link is cool, but it's a little hard to understand what it is given that it's not sorted.